eng/pipelines/templates/stages/vmr-post-build.yml

```diff
@@ -0,0 +1,67 @@
+parameters:
+# Branch of the VMR to use (to push to for internal builds)
+- name: vmrBranch
+  type: string
+  default: $(Build.SourceBranch)
+
+- name: pool_Windows
+  type: object
+  default:
+    name: $(defaultPoolName)
+    image: $(poolImage_Windows)
+    demands: ImageOverride -equals $(poolImage_Windows)
+    os: windows
+    
+stages:
+- stage: VMR_Post_build
+  displayName: VMR Post Build
+  dependsOn: VMR_Vertical_Build
+  variables:
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+    - group: Publish-Build-Assets
+    - name: azureDevOpsOrg
+      value: dnceng
+    - name: azureDevOpsProject
+      value: internal
+  - ${{ else }}:
+    - name: azureDevOpsOrg
+      value: dnceng-public
+    - name: azureDevOpsProject
+      value: public
+    - name: dn-bot-dnceng-build-rw-code-rw
+      value: ''
```

Comment:
I'm kinda surprised this works as blank, but I like it if it does. Otherwise System.AccessToken would potentially work.